release-notes/v4-tucker/4.7.12.md

```diff
@@ -0,0 +1,11 @@
+---
+title: 4.7.12
+---
+
+# 4.7.12
+
+12/08/2025
+
+- Warn users if they are running HarperDB in an application folder or root folder to prevent potential issues
+- Update `listSSHKeys` unit tests for better coverage
+- Remove unused process management functions
```

Comment:
Similarly, this seems like an internal detail not relevant for user-facing release notes.